Makefile

```diff
@@ -0,0 +1,3 @@
+.PHONY: graphql
+gql:
+	go build ./graphql/main.go
```

Comment:
Do we want this to run? Unsure 🤷‍♀️ .

Reply:
I just started it this way cause my personal flow has always been "build then run" and very rarely using `go run`, but I think it's perfectly fine to have a run option as well.